website/data/cli-nav-data.json

```
@@ -256,6 +256,16 @@
       }
     ]
   },
+  {
+    "title": "Accessing Metadata",
+    "routes": [
+      { "title": "Overview", "path": "metadata" },
```

```suggestion
      { "title": "Overview", "path": "/cli/commands/metadata" },
```
Might fix the build issue?